source/api/ut.pkb

```diff
@@ -267,13 +279,16 @@ create or replace package body ut is
       a_client_character_set
     );
     if l_reporter is of (ut_output_reporter_base) then
-      l_lines := treat(l_reporter as ut_output_reporter_base).get_lines_cursor();
+      l_data := treat(l_reporter as ut_output_reporter_base).get_lines_cursor();
       loop
-        fetch l_lines into l_line;
-        exit when l_lines%notfound;
-        pipe row(l_line);
+        fetch l_data into l_clob, l_item_type;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This FETCH INTO should be replaced by a FETCH BULK COLLECT INTO. ([plsql:UseFetchBulkCollectIntoCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AUseFetchBulkCollectIntoCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=809&issues=AWgA5Ioj_MPaBDn0kbLu&open=AWgA5Ioj_MPaBDn0kbLu)

[ISSUE_KEY]: AWgA5Ioj_MPaBDn0kbLu